### ISSUE TITLE
выключается после обновления страницы, пропадают связи файлов

### ISSUE THREAD

**Issue by tsymbal** [livestyle/issues#132]
Собственно всё в названии темы. 
Стоит рефрешнуть страницу, и плагин выключается, а после включения приходится настраивать все связи заново. 
Такой баг в Sublime 2 и 3. Обновление плагина и браузера не помогло. 
Раньше такого не было, пару месяцев назад появилось. 
Работаю по ftp с реальными сайтами на реальных серверах.

**Comment by sergeche** [comment]
Да, есть такая проблема, что-то случилось в последних версиях Хрома, начал терять соединения с некоторыми страницами. У меня в процессе новая версия, там должна быть исправлена эта проблема. Пока не знаю, когда выйдет, но на днях планирую вернуться к ней

**Comment by tsymbal** [comment]
Спасибо за ответ. Очень жду. Прям готов "контрибутировать" денег (несколько сотен рублей) в этот плагин. :)

**Comment by sergeche** [comment]
Спасибо :) Сможете потом поддержать проект через краудфандинговую кампанию ;)

**Comment by sergeche** [comment]
Кстати, мне было бы очень полезно услышать отзыв о самой работе LiveStyle, в основном интересует что сейчас не удобно сделано и что хотелось бы улучшить?
